extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -178,14 +178,6 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 			this.currentBook = book;
 		}
 		this._bookViewer = this._apiWrapper.createTreeView(this.viewId, { showCollapseAll: true, treeDataProvider: this });
-		this._bookViewer.onDidChangeVisibility(e => {
```

Comment:
Do we have to delete this code? This is the code that will set the highlight when the notebooks viewlet is visible.

Reply:
added it back (sorry I totally missed why it was added in the first place)